build-scripts/gulp/translations.js

```diff
@@ -33,7 +38,12 @@ gulp.task(
 
 // Panel translations which should be split from the core translations.
 const TRANSLATION_FRAGMENTS = Object.keys(
-  require("../../src/translations/en.json").ui.panel
+  JSON.parse(
+    readFileSync(
```

Comment:
Why not `readJsonSync`?

Reply:
I was trying not to add `fs-extra` where it didn't already exist.  It's CJS only so you have to import the whole thing which slows things down.